spec/fixture_spec.rb

```diff
@@ -11,6 +11,13 @@
       let(:other) { Licensee::License.find('other') }
       let(:none) { Licensee::License.find('none') }
       let(:expectations) { fixture_licenses[fixture] || {} }
+      let(:expected_key) {
```

Comment:
Annoying style complaint causing CI to fail here:

spec/fixture_spec.rb:14:26: C: [Correctable] Style/BlockDelimiters: Avoid using {...} for multi-line blocks.